### DISCUSSION TITLE
Where does freac expect to find encoders on a Linux/MacOS distribution?

### DISCUSSION THREAD

**Discussion by DavidPesticcio** [enzo1982/freac#659] [Help & Support]
Hi @enzo1982, I'm currently looking into resolving an issue with a [NixOS](https://nixos.org/) package of [freac](https://github.com/NixOS/nixpkgs/blob/nixos-24.11/pkgs/by-name/fr/freac/package.nix#L33), that currently shows no encoders in the [meh!](https://github.com/NixOS/nixpkgs/issues/196234) dialog.

Does freac expect them to be available on the `$PATH`, i.e. they are executable, or at a specific path, like `/usr/lib/` etc, because they are a library file?

Some insight from you to help diagnose the problem would be most appreciated. :-)

EDIT. I've found the `build-appimage` script in the `.github` directory which is helping somewhat, but would still appreciate your input.

**Comment by mastermines1** [comment]
Have any luck yet? Was thinking at looking at this problem.

**Reply by DavidPesticcio** [reply]
@mastermines1 - Yes, it works, but still WIP and not done in an entirely idiomatic NixOS way - feedback welcome! :smiley: 

https://github.com/NixOS/nixpkgs/pull/387158

**Comment by DavidPesticcio** [comment]
@enzo1982 - Can you help us with resolving this issue please? :smiley: :pray:

https://github.com/NixOS/nixpkgs/pull/387158#issuecomment-3184054955

Trying to achieve the same outcome as here: https://github.com/enzo1982/BoCA/issues/5
